src/app/components/LeftMenu/PartyCardList/index.tsx

```diff
@@ -16,11 +18,33 @@ const Box = styled.div`
     background: transparent;
 `;
 
+function score(party, search){
+    var score = 0
+    if(party.title.toLowerCase().includes(search.toLowerCase())){
```

Comment:
tolowercase filter is not enough

need to ignore trailing spaces

Reply:
maybe make seperate function

Reply:
ok I'll change this one!